packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -469,6 +473,9 @@ RouterService.reopen(Evented, {
      * `blog.index` when you visit `/blog`
      * `blog.post` when you visit `/blog/some-post-id`
 
+     This property is updated after Route's [didTransition](/ember/release/classes/Route/events/didTransition?anchor=didTransition)
+     action and before `routeDidChange` being fired.
```

Comment:
FWIW, I would actually prefer to change these to actually be based on `routeDidChange`.